pootle/core/views.py

```diff
@@ -48,6 +48,9 @@
 from .utils.json import PootleJSONEncoder, jsonify
 
 
+EXPORT_VIEW_QUERY_LIMIT = 10000
```

Comment:
Wondering if this should be a POOTLE_* config variable so its more easily adjustable?